inc/Utf8/tables/romanization.php

```diff
@@ -298,9 +298,6 @@
 
     // 3 character syllables, っ doubles the consonant after
     'っちゃ' => 'ccha',
-    'っちぇ' => 'cche',
-    'っちょ' => 'ccho',
-    'っちゅ' => 'cchu',
```

Comment:
If I search for `っちぇ` in the file, I got another hit that looks different `ッチェ`, but by the ctrl-F of the browser was the same. I wonder if this are exactly the same items, or that there are minor differences? As they look different, I guess they are different.

Is there a Japanese speaker that can shine some light on these examples?

On the other hand, if the utf8 functions handle these characters the same as rector, then this cleanup makes sense of course for how it is used.

Reply:
@ssahara might have some input? Basically the romanization tables for Japanese had some double entries for years. with the later ones overwriting the former ones. This change removed the duplicates so there should be absolutely no change in behaviour. But it would be interesting to know if the remaining entries are reasonable or if the removed one would have been the better pick all those years...

Reply:
I moved this to it's own issue: https://github.com/dokuwiki/dokuwiki/issues/4047

Reply:
BTW. CTRL-F is not a reliable way to test this issue as the browser is doing some kind of fuzzy matching for non-latin chars.

Reply:
Of course, but it was to indicate that there are more tools that consider these characters as equal, while not looking equal.